set.mm

```diff
@@ -420312,6 +420323,35 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        </p>
 
        <ul>
+       <li><b>Contributors.</b>
+       The (original) contributor of a theorem is the first one who proved the
```

Comment:
Well, you don't prove an axiom or definition, but it still carries a contributed tag.

Reply:
Maybe "The (original) contributor of an assertion (theorem, definition or axiom) is the first one who proved/provided the assertion ..." would be better?

Reply:
I think that is exact venough

Reply:
Sentence revised accordingly.